schema/spacewalk/upgrade/susemanager-schema-5.0.8-to-susemanager-schema-5.0.9/300-replace-susePackageExcludingPartOfPtf-view.sql

```diff
@@ -0,0 +1,34 @@
+ALTER TABLE rhnPackage
+    ADD COLUMN IF NOT EXISTS is_ptf
+        NUMERIC(1)
+	DEFAULT (0)
+	CONSTRAINT rhn_package_is_ptf_check
+	    CHECK (is_ptf in ( 1 , 0 )),
+    ADD COLUMN IF NOT EXISTS is_part_of_ptf
+        NUMERIC(1)
+        DEFAULT (0)
+        CONSTRAINT rhn_package_is_part_of_ptf_check
+            CHECK (is_part_of_ptf in ( 1 , 0 ));
+
+CREATE INDEX IF NOT EXISTS rhn_package_is_ptf_idx ON rhnPackage (is_ptf);
+CREATE INDEX IF NOT EXISTS rhn_package_is_part_of_ptf_idx ON rhnPackage (is_part_of_ptf);
```

Comment:
These indexes are missing in the original definition files.

Reply:
Added!